src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitReportMapper.java

```diff
@@ -127,6 +128,7 @@ private ReporterModel forCourcirc(@NonNull final ReporterModel reporterModel) {
         for (final Report report : reporterModel.getReports()) { //we modify logs conditionally here
             if (StringUtils.endsWith(report.getDefaultMessage(), PATTERN_TRANSIENT_REACTANCE_TOO_LOW)) { //we match line "X.ABCDEF1 : transient reactance too low ==> generator ignored"
                 logsTransientReactanceTooLow.add(StringUtils.removeEnd(report.getDefaultMessage(), PATTERN_TRANSIENT_REACTANCE_TOO_LOW).trim()); //... to get the node name
+                copyReportAsTrace(newReporter, report);
```

Comment:
update comment line 117

Reply:
Need to change "reports that are [...] replaced  by a single aggregate report" they are no longer replaced after this pr

Reply:
ok